### ISSUE TITLE
Feature Request: Add a icon for thesis

### ISSUE THREAD

**Issue by WeiViming** [chrisgrieser/alfred-bibtex-citation-picker#36]
### Feature Requested

Hello, is it possible to add a icon for `@phdthesis` and `@mastersthesis`? Thx.

```
@phdthesis{Zohner_2017_Faster,
  title = {Faster Oblivious Transfer Extension and Its Impact on Secure Computation},
  author = {Zohner, Michael},
  year = {2017},
  address = {Darmstadt},
  url = {https://tuprints.ulb.tu-darmstadt.de/6154/},
  urldate = {2023-07-03},
  copyright = {CC BY 4.0 International - Creative Commons, Attribution},
  language = {en},
  school = {Technische Universit\"at}
}

@mastersthesis{Bicer_2017_Efficiency,
  title = {Efficiency Optimizations on Yao's Garbled Circuits and Their Practical Applications},
  author = {Bi{\c c}er, Osman},
  year = {2017},
  eprint = {1703.03473},
  address = {\.Istanbul, Turkey},
  url = {http://arxiv.org/abs/1703.03473},
  urldate = {2021-08-17},
  archiveprefix = {arxiv},
  language = {en},
  school = {\.Istanbul \c{S}ehir University}
}
```

### Relevant Screenshot

<img width="718" alt="image" src="https://github.com/chrisgrieser/alfred-bibtex-citation-picker/assets/35755899/8565def9-74f7-4f08-a13c-86a515f7d8a9">

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by chrisgrieser** [comment]
good point. Changed in release 11.5.2

**Comment by WeiViming** [comment]
Sorry, it doesn't show in release 11.6.2.

**Comment by chrisgrieser** [comment]
Yeah I botched sth there, just get the latest release

**Comment by WeiViming** [comment]
Sorry, it doesn't work in the latest release. I've already reinstalled this workflows.
<img width="700" alt="image" src="https://github.com/chrisgrieser/alfred-bibtex-citation-picker/assets/35755899/08c16bad-5391-4340-99c6-3b57494dcee7">

**Comment by chrisgrieser** [comment]
sorry, fixed with 11.6.4
